backend/src/Entity/ImageEntity.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\ImageEntityRepository;
+use Doctrine\ORM\Mapping as ORM;
+
+#[ORM\Entity(repositoryClass: ImageEntityRepository::class)]
+class ImageEntity
+{
+    #[ORM\Id]
+    #[ORM\GeneratedValue]
+    #[ORM\Column(type: 'integer')]
+    private $id;
+
+    #[ORM\Column(type: 'string', length: 255)]
+    private $imagePath;
+
+    #[ORM\Column(type: 'string', length: 255)]
+    private $entityType;
```

Comment:
- change type to integer.
- we will use numbers as values illustrated in consts class.